packages/react-components/src/components/AttachmentUploadCards.tsx

```diff
@@ -48,11 +48,11 @@ export interface ActiveFileUpload {
 }
 
 /**
- * Strings of _FileUploadCards that can be overridden.
+ * Strings of _AttachmentUploadCards that can be overridden.
  *
  * @internal
  */
-export interface _FileUploadCardsStrings {
+export interface _AttachmentUploadCardsStrings {
   /** Aria label to notify user when focus is on cancel file upload button. */
   removeFile: string;
```

Comment:
```suggestion
  removeAttachment: string;
```

Reply:
i am not sure, apparently this name has been GA-ed? 

<img width="612" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/2858f271-f9ee-403e-a03b-093d8eb55d22">